Describe "Animals in Motion" track 2026 (#200)
neuroinformatics-unit/neuroinformatics-unit.github.io: docs/source/open-software-summer-school/2026/animals-in-motion.md

* **Motion track analysis:** a hands-on tutorial of [movement](https://movement.neuroinformatics.dev)—a Python package for cleaning, visualising, and analysing motion tracks output by [SLEAP](https://sleap.ai/), [DeepLabCut](https://deeplabcut.org/) and similar tools. This part includes coding exercises and case studies on applying motion quantification to real-world scientific questions.
* **Behaviour segmentation:** a primer on decomposing continuous motion into discrete behaviours, followed by a practical tutorial of a supervised behaviour segmentation tool.

### Collaboration days

The final two days—**Thursday and Friday**—are dedicated to collaboration. We will join forces with participants from the **Large Array Data** track to work together on participant-led projects.

* **Skill building:** we'll start with a practical workshop on **Git and GitHub** to equip everyone with the necessary skills for collaborative coding.

> * **Project-based work:** participants will self-organize into small teams to tackle projects hands-on. **Coding is not a requirement**; any idea that benefits from collaboration with other attendees is welcome. Potential project ideas include, but are not limited to:

Reviewer: ```suggestion
* **Project-based work:** participants will self-organise into small teams to tackle projects hands-on. **Coding is not a requirement**; any idea that benefits from collaboration with other attendees is welcome. Potential project ideas include, but are not limited to:
```

    * *Apply a tool:* use any learned software to analyze a new dataset (your own or a public one).
    * *Give feedback:* report bugs and suggest features by raising issues on relevant open-source tools.
    * *Make a contribution:* submit a pull request to an open-source repository (support will be provided).
    * *Collaborative writing:* draft a white paper, blog post, or documentation together.

>     * *Prototype an idea:* test a cool new analysis or method on real-world data.

Reviewer: ```suggestion
    * *Prototype an idea:* Experiment with a new analysis or method.
```
